StyleCop.Analyzers/StyleCop.Analyzers.Test/ReadabilityRules/SA1130UnitTests.cs

```diff
@@ -880,5 +880,57 @@ public class TestClass
 
             await VerifyCSharpFixAsync(testCode, expected, fixedCode, CancellationToken.None).ConfigureAwait(false);
         }
+
+        [Fact]
+        [WorkItem(3279, "https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3279")]
+        public async Task TestDelegateUsedAsSecondNamedArgumentAsync()
+        {
+            var testCode = @"
+using System;
+using System.Linq;
+public class TypeName
+{
+    public void Test()
+    {
+        string[] capture = new string[] { ""test"" };
+        Test2(resolve: delegate
```

Comment:
💡 Can you add a test showing an explicit named argument, where the name doesn't match any parameter? This would be an error, but we still want to make sure the analyzer doesn't throw an exception.

Reply:
@sharwell Not sure if I understand: using parameter name which doesn't fit method signature _should_ produce CS1739, shouldn't it? Is your intention to test that CS1739 is raised?

Reply:
Compiler errors prevent the build from succeeding, but the analyzers still run. We want to make sure the analyzer isn't going to fail for this case.